test/remote-connector.test.js

```diff
@@ -87,3 +87,81 @@ describe('RemoteConnector', function() {
     });
   });
 });
+
+describe('Custom Path', function() {
+
+  var test = this;
+
+  before(function(done) {
+
+    test.server = server = loopback();
+    server.set('legacyExplorer', false);
+    server.use('/api', loopback.rest());
+    server.set('host', '127.0.0.1');
+    server.set('port', 3001);
```

Comment:
Please use an ephemeral port (`0`) to prevent clashes with other apps/tests that may be already listening on `3001`. see the test above for how to achieve that.

It will be best to extract the duplicated code between this setup and the setup in the `describe` block above to a shared method(s).